.appveyor.yml

```diff
@@ -6,6 +6,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
+  - npm install -g npm
```

Comment:
Same as we do in Travis, this forces npm 3 to get deduping even on older versions of Node.